OpenGL/Constructs/VBO.cs

```diff
@@ -70,7 +113,25 @@ private set
         /// Specifies the number of instances that will pass between updates of the generic attribute slot.
         /// Only used for instance drawing.
         /// </summary>
-        public uint Divisor { get; private set; }
+        public uint Divisor { get; set; } = 0;
+
+        /// <summary>
+        /// Specifies whether fixed-point data values should be normalized (true) or converted directly as fixed-point values (false) when they are accessed.
+        /// If normalized is set to true, it indicates that values stored in an integer format are to be mapped to the range [-1,1] (for signed values) 
+        /// or [0,1] (for unsigned values) when they are accessed and converted to floating point. 
+        /// Otherwise, values will be converted to floats directly without normalization.
+        /// </summary>
+        public bool Normalize { get; set; } = true;
+
+        /// <summary>
+        /// Specifies whether types, other than float, should be cast to float.
+        /// </summary>
+        public bool CastToFloat { get; set; } = true;
+
+        /// <summary>
+        /// Specifies whether the VBO contains an integral type.
+        /// </summary>
+        public readonly bool IsIntegralType;
```

Comment:
I would suggest changing this to something like:

```
private readonly bool isIntegralType;

/// <summary>
/// Specifies whether the VBO contains an integral type.
/// </summary>
public bool IsIntegralType { get { return isIntegralType; } }
```

I think C# 6 even has an auto-property for this, so the following would work as well:

`public bool IsIntegralType { get; }`

This avoids exposing a field.